### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#52674]
#### VFS team name
Sitewide Content

#### VFS product name
Enrollment verification

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
The alert heading "Sign in to verify your school enrollment" is an H1.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
From context I think this should be an H3 instead.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by katherine-fung** [comment]
Hi @btruongafs, could you have your team look at this and make the widget heading "Sign in to verify your school enrollment" into an H3? 

Can you please tag me here when the change is live? Thanks!

**Comment by RLHecht** [comment]
@btruongafs Checking in on this. Is this still in progress? Thanks!

**Comment by btruongafs** [comment]
@RLHecht we have closed down our development for the product launch. This issue will remain open and will be address post-launch.

**Comment by joshkimux** [comment]
@btruongafs Hi Bobby ✨ 

Checking in on this again-- [platform recommends fixing `defect-2` (serious accessibility) issues within 1-2 sprints](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-508-defect-2) post launch. Has there been any progress on this?

**Comment by katherine-fung** [comment]
Hi @coforma-terry. 

The header in question here ("Sign in to verify your school enrollment") is in a widget. The content team can't make any updates to widget. Only @btruongafs and the product team can make the necessary change. I'll check in with their team on slack.

**Comment by RLHecht** [comment]
@btruongafs checking if you have a ticket for your team to work on this?
